### DISCUSSION TITLE
Multiple IDs are added to different address books, the aliases are synchronized, and the aliases are displayed synchronously in the device list.

### DISCUSSION THREAD

**Discussion by Rocky-Jinxing** [rustdesk/rustdesk-server-pro#376] [General]
When multiple IDs are added to different address books, I think it is possible to bind a unique alias and password to this ID, and the alias can also be seen in the device list, and then added to different address books.

Problem: If it is added to different address books as it is, and then the device is no longer used or the password is changed, it is necessary to delete each device ID in the previously added address book, which is too cumbersome and easy to miss.

**Comment by rustdesk** [comment]
Will be considered, but low priority and need to get more suggesion  from other users.

**Reply by Rocky-Jinxing** [reply]
> Will be considered, but low priority and need to get more suggesion from other users.

@rustdesk The aliases in the address book cannot be synchronized to the device list in the web backend. Over time, when we have a large number of controlled devices, we cannot control which enabled controlled devices are being used by us, which will waste our controlled device license. We have now enabled 180 controlled terminals, and we cannot verify them one by one through device IDs.

**Reply by rustdesk** [reply]
Actually the other `Pro` users use this for your purpose. Alias here is not for this purpose, our alias is defined for different users can give different alias name to the same device, so it is impossible to sync to device list.

![image](https://github.com/user-attachments/assets/6e8c4a33-ca18-43d5-b3e4-569e422b1052)

**Reply by Rocky-Jinxing** [reply]
> Actually the other `Pro` users use this for your purpose. Alias here is not for this purpose, our alias is defined for different users can give different alias name to the same device, so it is impossible to sync to device list.
> 
> ![image](https://private-user-images.githubusercontent.com/71636191/363100153-6e8c4a33-ca18-43d5-b3e4-569e422b1052.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjUwMDcxNzYsIm5iZiI6MTcyNTAwNjg3NiwicGF0aCI6Ii83MTYzNjE5MS8zNjMxMDAxNTMtNmU4YzRhMzMtY2ExOC00M2Q1LWIzZTQtNTY5ZTQyMmIxMDUyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MzAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODMwVDA4MzQzNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWE1NWE5YzAyN2ZhYmU1MmNhMTg4ODBmNWVhMDQ0MTFhMDE3NjlkM2MxOWRjNjQ5NzIzMWFkMmZkMGUwYWU1NzUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.8Y3eO3_8Lvk6yuHYbKb0UFqSMoZ5RtA2x8mDc5NQxcQ)

@rustdesk How do we find out if the devices are already enabled and not in the address book? We need to clean up these devices. Save our license

**Reply by rustdesk** [reply]
Good point, we can consider this, but apologize, we can not give your an ETA. There are so many in to-do list.
